app/controllers/audits_controller.rb

```diff
@@ -90,6 +90,12 @@ def destroy
 
   private
 
+  def handle_storage_location_error
+    if @audit.errors.present?
+      flash[:error] = "Storage location must be selected."
```

Comment:
Are we sure that there is only one possible error for audits, and it's if storage location is blank? I think we probably should clean up the error(s) we get rather than assuming we know what it is.

Reply:
@dorner - So, I was playing around with the errors that could possibly come up with creating an audit, and the only error that I was able to produce with any combination of actions, was the storage location error. The problem that I encountered with collecting the errors, was that multiple errors were coming through, and displaying those does not fix the repetitiveness issue.